src/nodes.coffee

```diff
@@ -1223,18 +1223,128 @@ exports.Class = class Class extends Base
     klass = new Assign @variable, klass if @variable
     klass.compileToFragments o
 
+#### Import and Export
+
+exports.Module = class Module extends Base
+  constructor: (@type, @clause, @moduleName, @default = no) ->
+    if @type isnt 'import' and @type isnt 'export'
+      @error 'module type must be import or export'
+
+    if @moduleName? and @moduleName instanceof StringWithInterpolations
+      @error 'the name of the module to be imported from must be an uninterpolated string'
+
+    if @type is 'export' and @default is no and @clause instanceof Class and not @clause.variable?
+      @error 'anonymous classes cannot be exported'
+
+  children: ['clause', 'moduleName']
+
+  isStatement: YES
+  jumps:       THIS
+  makeReturn:  THIS
+
+  compileNode: (o) ->
+    if o.indent.length isnt 0
+      @error "#{@type} statements must be at top-level scope"
```

Comment:
Good point re backticks, maybe that’s the best solution. My question was just about line 1246 above, using `o.indent.length` as a check to determine if we’re in an `export` statement itself as opposed to a block (expression) within the `export` statement, e.g. the body of the `foo` function in `export default foo ->`. Using `o.indent` works, assuming we never cave to Meteor’s desired syntax and `export` statements must always begin at the start of a line, but it feels brittle to me and a bit indirect. I feel like there should be some way of explicitly getting the “parent” of a block and seeing if that’s an `export`, rather than counting indentation characters.